NOTICE

```diff
@@ -0,0 +1,5 @@
+Neon
+Copyright 2022 Neon Inc.
+
+The PostgreSQL submodule in vendor/postgres is licensed under the
+PostgreSQL license. See vendor/postgres/COPYRIGHT.
```

Comment:
Let's have a blurp here, saying something like "this is licensed under the Apache license". Best to copy-paste that from somewhere, to use some common phrasing of that.

Reply:
Hmm, I did not see anything like this in Apache's examples like https://infra.apache.org/licensing-howto.html

Feel free to alter as you deem necessary and merge. I believe the `licensee` library does not care about `NOTICE` at all.